mypy/suggestions.py

```diff
@@ -437,7 +437,7 @@ def get_suggestion(self, mod: str, node: FuncDef) -> PyAnnotateSignature:
         if self.no_errors and orig_errors:
             raise SuggestionFailure("Function does not typecheck.")
 
-        is_method = bool(node.info) and not node.is_static
+        is_method = not node.is_module_level_func and not node.is_static
```

Comment:
Let's leave this as it was, since using an `info` attribute as a boolean is a common idiom.